tests/wpt/web-platform-tests/css/cssom/CharacterData-replaceData.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
```

Comment:
This needs a `<link rel="match">`, a `<title>`, and if you want also a `<link rel="author">` (or two, if you also want to include me, as you wish).

The reference maybe also could get a `<title>` at least.

Also, what do you think about naming the file something like `stylesheet-replacedata-dynamic.html` or something like that? Not a big deal if not.

Reply:
Great! Thanks for pointing out why these tests got errors.
I'd like to change the filename like what you said. I'll change it.
Besides, please let me add you as `author` as well :)